astropy/coordinates/builtin_frames/intermediate_rotation_transforms.py

```diff
@@ -22,17 +22,18 @@
 
 # # first define helper functions
 def gcrs_to_cirs_mat(time):
-    #felestial-to-intermediate matrix
-    return erfa.c2i06a(time.jd1, time.jd2)
+    #celestial-to-intermediate matrix
+    return erfa.c2i06a(time.tt.jd1, time.tt.jd2)
 
 def cirs_to_itrs_mat(time):
     #compute the polar motion p-matrix
     xp, yp = get_polar_motion(time)
```

Comment:
While you are at it, could you adjust `get_polar_motion` to just pass on the `time` instance to the `iers` table? It does `time.utc` internally (alternatively, pass on `time.utc` here, but it seems better to just do it where it should be done).